library/blivet.py

```diff
@@ -503,7 +532,7 @@ def _create_members(self):
             self._blivet.format_device(member, self._get_format())
             members.append(member)
 
-        if use_partitions:
+        if use_partitions and self._pool.get('raid_level') in [None, "null", ""]:
```

Comment:
It looks like it might be worthwhile to add a property called `is_raid` or similar to streamline this check since it's repeated several times.

Reply:
Fixed